mwana/apps/export/views.py

```diff
@@ -0,0 +1,132 @@
+# vim: ai ts=4 sts=4 et sw=4
+# Create your views here.
+import psycopg2
+from mwana import localsettings as settings
+from django.http import HttpResponse
+import json
+from mwana.apps.locations.models import Location
+from collections import OrderedDict
+
+
+def get_turnaround(request):
+    '''
+    key_value = ""
+    if ((request.method == 'GET') and ('key_value' in request.GET)):
+        key_value = request.GET['key_value']
+    '''
+    connection = psycopg2.connect(host=settings.DATABASES['default']['HOST'], database=settings.DATABASES['default']['NAME'],\
+                    user=settings.DATABASES['default']['USER'], password=settings.DATABASES['default']['PASSWORD'])
+    
+    districts = Location.objects.filter().distinct()
+    
+    site_info_req = OrderedDict()
+
+    try:
+        cursor = connection.cursor()
+        cursor.execute(SQL_QUERY_SELECT) 
+        site_info_req = dictfetchall(cursor)
+
+    except:
+        log = "oops...", sys.exc_info()
+    finally:
+        connection.close()
+        return HttpResponse(json.dumps(site_info_req))   
+
+def dictfetchall(cursor):
+    # "Returns all rows from a cursor as a dict"
+    to_return_dict = []
+    desc = cursor.description
+    if cursor.rowcount > 0:
+        o_dict_rows = [
+            OrderedDict(zip([str(col[0]) for col in desc], row))
+            for row in cursor.fetchall()
+        ]
+
+        for row in o_dict_rows:
+            o_dict_str = OrderedDict()
+            for key,val in row.items():
+                o_dict_str[key] = str(val)
+
+            to_return_dict.append(o_dict_str)
+    else:
+        cols = [desc[0] for desc in cursor.description]
+        dict_cols = OrderedDict()
+        for col in cols:
+            dict_cols[col] = ''
+        to_return_dict.append(dict_cols)
+
+    return to_return_dict
+
+SQL_QUERY_SELECT = '''
```

Comment:
Query seems to run fast, but can we use true median? In some cases the differences are quite significant if you use median